nix/hosts/webforge/configuration.nix

```diff
@@ -8,5 +8,32 @@
   networking.hostName = "webforge";
   networking.domain = "tahoe-lafs.org";
 
+  # Enable firewall with the required ports
+  networking.firewall = {
+    enable = true;
+    allowPing = false;
+    allowedTCPPorts = [
+      80
+      443
+    ];
+    allowedUDPPorts = [
+      443
```

Comment:
QUIC is an overly complicated and complex abomination IMHO, but also OK to me if you want it :)

Reply:
I've tuned the config to be that citizen. ThX